### DISCUSSION TITLE
RESTful API for simple seek forward/backward for Foobar2000

### DISCUSSION THREAD

**Discussion by miaLDN** [hyperblast/beefweb#259] [Ideas]
Hi,

It would be amazing to have a method to simply seek forward/backward X seconds of a track.

Foobar2000 has main->Playback->Seek->[Back|Ahead][1s|5s|10s|30s|60s|120s|300s|600s]

I listen to podcasts and often want to rewind, or skip forward.

There is `GET /player` for player-state, and `POST /player`, which is possible to use, but requires knowing, volume setting, units, current play position, etc.

An API for something like `POST /seek` would be awesome :)  Then I can easily dedicate two buttons on my Stream Deck to Seek Back and Seek Ahead.

**Comment by hyperblast** [comment]
Hi,

`POST /player` already supports that.

For example for seeking forward for 5s you can use `relativePosition` property in the request body:

```json
{"relativePosition":5}
```

All `POST /player` parameters are optional so you can specify only those you want to change.
